api/envoy/data/tap/v2alpha/transport.proto

```diff
@@ -47,7 +47,7 @@ message Event {
   }
 }
 
-// Sequence of read/write events that constitute a captured trace on a socket.
+// Sequence of read/write events that constitute a trace on a socket.
 // Multiple Trace messages might be emitted for a given connection ID, with the
 // sink (e.g. file set, network) responsible for later reassembly.
 message Trace {
```

Comment:
You're probably going to want to move from a "entire connection trace" to a chunked trace for streaming in a later PR.

Reply:
Yeah I'm going to be working on the streaming stuff in later PRs for sure.